### DISCUSSION TITLE
Remove instrument response

### DISCUSSION THREAD

**Discussion by sujan1746** [ROBelgium/MSNoise#264] [Q&A] [answered]
I used `remove_response=N` because my study area has only one type of instruments.

But now I run MSNoise separately about another study area.

So I need to remove response file because the instruments which installed in the area are two types.

There are two questions about removing instruments.

1. Response_format
In the config, I set as below.
![image](https://user-images.githubusercontent.com/89066297/152539804-8a043c6a-d95e-4236-82cf-6316b346d563.png)

What I wonder is whether there is a way to confirm the waveform before and after removing response file?

2. Stations.csv
This is my stations.csv file.
![image](https://user-images.githubusercontent.com/89066297/152547244-44991e79-e255-46d6-8e72-afabb99673e0.png)

I have three instrument codes (HHZ, HHE, HHZ).

In the table, how can I apply all instruments when computing cc? 

I edit the instrument code as * or HH* but still occurs ` No instrument response information for net.sta..HHE`  and this same occurs related to HHN, HHZ.
                   
Edit..
I saw in MSNoise, the format of dataless or inventory is available, not the resp. I will remove it by using obspy and do the next process.

**Comment by sujan1746** [comment, marked as answer]
I have RESP and I thought it would be not bad to remove response file first and run MSNoise.

But it is not a good way because I need to convert from raw data to response_removed_data and save them every time when I add new data.

So I convert RESP to Station XML format by using below code.

```
`inv='RESP.net.sta..comp'
inv=read_inventory(inv)
inv.write("net.sta..comp.xml", format="stationxml", validate=True)`
```

And made instrument code in the stations.csv as * because I need to remove response about all components (e.g. HHZ, HHN) and made pre_filt as None for running faster.

I am not sure it is the right way to convert RESP file to Station XML file but any error shows up when computing CC.

I will check the waveform after all of the CC computation done.

**Reply by ThomasLecocq** [reply]
in msnoise you can use any format obspy can read actually

your way to create the XML is goo.
The "instrument" field in the station table is not used, but if you want to compute the other components, you have of course to scan them, and have them in "components_to_compute". Be super careful disabling pre_filt, it's really necessary in many many cases !!!

**Reply by sujan1746** [reply]
Yeah I add "ZZ,ZE,ZN,EZ,EE,EN,NZ,NE,NN" in "components_to_compute". But I did not know that the pre_filt is necessary. I add "(0.00005, 0.00006, 44.0, 49.0)" in the pre_filt because the sampling rate of the raw data is 100Hz and restart to compute CC. Thank you so much for letting me know!

**Reply by ThomasLecocq** [reply]
why so low frequencies ? Instrument responses can be very unstable/not defined at such low frequencies

**Reply by sujan1746** [reply]
Oh.... I see.. Then I would set as 0.001, 0.002. Thanks!
